dipy/viz/app.py

```diff
@@ -109,9 +107,14 @@ def __init__(self, tractograms=None, images=None, pams=None,
             small animal data you may need to use something smaller such
             as 2.0. The threshold is in mm. For this parameter to be active
             ``cluster`` should be enabled.
-        random_colors : bool
-            Given multiple tractograms have been included then each tractogram
-            will be shown with different color
+        random_colors : string
```

Comment:
`str, optional` 

Can you also add the optional missing on the parameter above. Thanks!

Reply:
Added